taxonomies/food/ingredients.txt

```diff
@@ -27681,6 +27681,11 @@ carbon_footprint_fr_foodges_ingredient:fr: Farine de blé
 carbon_footprint_fr_foodges_value:fr: 1.2
 ciqual_proxy_food_code:en: 9410
 ciqual_proxy_food_name:fr: Farine de blé tendre ou froment T110
+ecobalyse:en: flour
+ecobalyse_labels_en_organic:en: flour-organic
+ecobalyse_origins_en_france:en: flour
```

Comment:
If the value is the same as the default in ecobalyse:en: , I think it's best not to add properties like ecobalyse_origins_en_france:en: . We can add only the ones that are different from the default.